styles.css

```diff
@@ -235,4 +264,107 @@ body {
   .separa {
     width: 100%;
   }
+
+  .contactform {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+    gap: 50px;
+    background-color: #6070ff;
+    border-radius: 100px 0 0 0 / 100px;
+  }
+
+  .contactshapes {
+    position: absolute;
+    height: 100vh;
+    margin-left: 40vw;
+    padding-top: 8vh;
+  }
+
+  .cme {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    text-align: center;
+    margin-top: 100px;
+  }
+
+  .pme {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  #name {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    color: #cfd8dc;
+    width: 80vw;
+    height: 5vh;
+    margin-left: 8vw;
+  }
+
+  #email {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    color: #cfd8dc;
```

Comment:
[OPTIONAL] Kindly change the color of the input. It is supposed to be #172B4D according to the requirement. 😊 ☺️